.github/workflows/reusable-performance.yml

```diff
@@ -29,7 +29,8 @@ on:
     secrets:
       CODEVITALS_PROJECT_TOKEN:
         description: 'The authorization token for https://www.codevitals.run/project/wordpress.'
-        required: true
+        required: false
+        default: ''
```

Comment:
Makes sense to not require it, but we should then make sure that it _is_ required for the particular scenario where it's needed. Can you add a check to the relevant step so that it fails if `CODEVITALS_PROJECT_TOKEN` is not provided or empty?

Reply:
I've added a check in [b58da09](https://github.com/WordPress/wordpress-develop/pull/7550/commits/b58da0964abe44db3892bc3677486a1b4642aeaf) to log and fail if the value is not set. This should also alert us if the secret gets removed from the repo for some reason.